Assets/PatchKit Patcher/Scripts/AppData/Remote/Downloaders/TorrentClient.cs

```diff
@@ -101,10 +101,20 @@ public TorrentClientStatus GetStatus(CancellationToken cancellationToken)
             }
         }
 
+        private void ThrowIfProcessExited()
+        {
+            if (_process.HasExited)
+            {
+                throw new TorrentClientException(string.Format("torrent-client has exited with error code {0}", _process.ExitCode));
```

Comment:
The name should be more specific - `TorrentClientCrashException` or something like that.